disperser/apiserver/server.go

```diff
@@ -413,11 +433,13 @@ func (s *DispersalServer) checkRateLimitsAndAddRatesToHeader(ctx context.Context
 
 		globalRates, ok := s.rateConfig.QuorumRateInfos[param.QuorumID]
 		if !ok {
+			s.metrics.HandleInternalFailureRpcRequest("DisperseBlobAuthenticated")
```

Comment:
`checkRateLimits...` is called by both "DisperseBlob" and "DispserseBlobAuthenticated". Maybe we need to pass in the `apiMethod` argument here as elsewhere.

Reply:
Good catch, done